packages/spatial/src/physics/systems/PhysicsPreTransformSystem.ts

```diff
@@ -195,3 +195,11 @@ export const PhysicsPreTransformSystem = defineSystem({
   insert: { after: TransformDirtyUpdateSystem },
   execute
 })
+
+export const _PhysicsPreTransformFunctions = {
```

Comment:
Let's remove the underscore from this definition name. Even though these functions are unlikely to be used externally, we should keep the namespace clear.

Reply:
Fixed with 89c3e14